Readme.md

```diff
@@ -7,28 +7,44 @@
+**Key Features:**
+* Kubernetes native installation
+* Hundreds of default events
+* Ships with basic set of behavioral signatures for malicious detection out of the box 
+* Easy configuration through Tracee Policies 
+* Kubernetes native user experience that is targetted at cluster administrators
+
+> We release new features and changes on a monthly basis. Learn more about the letest release in our [discussions.](https://github.com/aquasecurity/tracee/discussions)
+
+To learn more about Tracee, check out the [documentation](https://aquasecurity.github.io/tracee/latest/docs/overview/). 
 
 ## Quickstart
 
-You can easily start experimenting with Tracee using the Docker image as follows:
+Either:
+* [Install Tracee in your Kubernetes cluster.](https://aquasecurity.github.io/tracee/latest/getting-started/kubernetes-quickstart)
+* [Experiment using the Tracee container image.](https://aquasecurity.github.io/tracee/latest/#quickstart)
+
+Steps to get started:
+1. [Install Tracee in your Kubernetes cluster through Helm](https://aquasecurity.github.io/tracee/latest/getting-started/kubernetes-quickstart/)
+2. Query logs to see detected events
+3. Filter events through [Tracee Policies](https://aquasecurity.github.io/tracee/latest/tutorials/k8s-policies/) 
+4. [Manage logs through Grafana Loki](https://aquasecurity.github.io/tracee/latest/tutorials/promtail/) or your preferred monitoring solution
+
+[The tutorials section showcases](https://aquasecurity.github.io/tracee/latest/tutorials/overview/) integration between monitoring solutions and Tracee.
 
-```console
-docker run \
-  --name tracee --rm -it \
-  --pid=host --cgroupns=host --privileged \
-  -v /etc/os-release:/etc/os-release-host:ro \
-  -v /boot/config-$(uname -r):/boot/config-$(uname -r):ro \
-  aquasec/tracee:$(uname -m)
-```
+![Example log output in Tracee pod](./docs/images/log-example.png)
+Example log output in Tracee pod
+## Contributing
+  
+Join the community, and talk to us about any matter in the [GitHub Discussions](https://github.com/aquasecurity/tracee/discussions) or [Slack](https://slack.aquasec.com).  
+If you run into any trouble using Tracee or you would like to give use user feedback, please [create an issue.](https://github.com/aquasecurity/tracee/issues)
 
-To learn how to install Tracee in a production environment, [check out the Kubernetes guide](https://aquasecurity.github.io/tracee/latest/getting-started/kubernetes-quickstart).
+Find more information on [contributing to the source code](https://aquasecurity.github.io/tracee/latest/contributing/overview/) in the documentation.
 
----
+## More about Aqua Security
 
 Tracee is an [Aqua Security](https://aquasec.com) open source project.  
-Learn about our open source work and portfolio [Here](https://www.aquasec.com/products/open-source-projects/).  
-Join the community, and talk to us about any matter in [GitHub Discussion](https://github.com/aquasecurity/tracee/discussions) or [Slack](https://slack.aquasec.com).  
```

Comment:
Don't you think it's a good thing to have a link to our slack channel?

Reply:
Yes, I have added it above -- just moved it around a bit